x-pack/plugins/apm/public/components/shared/charts/most_used_chart/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import type { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+import { ViewMode } from '@kbn/embeddable-plugin/public';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { ApmPluginStartDeps } from '../../../../plugin';
+import { getLensAttributes } from './get_lens_attributes';
+
+export enum MostUsedMetric {
+  DEVICE_NAME = 'device.model.name',
+  NCT = 'network.connection.type',
+  APP_VERSION = 'agent.version',
+  OS_VERSION = 'host.os.version',
+}
+
+export function MostUsedChart({
+  start,
+  end,
+  filters,
+  metric,
+  bucketSize = 5,
```

Comment:
Since the default bucket size is always used, it could be defined as a `const` in `getLensAttributes` and add it as a prop later on if needed.

Reply:
done here: https://github.com/elastic/kibana/pull/144232/commits/482d2fcaa24505bc00f74824c8e8a21502c2922c